core/src/main/java/com/arcrobotics/ftclib/hardware/motors/MotorGroup.java

```diff
@@ -27,8 +34,9 @@ public MotorGroup(Motor... motors) {
      */
     @Override
     public void set(double speed) {
-        for (Motor x : group) {
-            x.set(isInverted ? -speed : speed);
+        group[0].set(speed);
```

Comment:
This change in behavior seems to be a breaking change.

Reply:
This I would consider a breaking change on the grounds that it changes the behavior of the motor group. Although I think our definitions of breaking change are different. When I say something is a breaking change, I mean that the user would have to modify their program to fit the changes made when bumping to the latest version (i.e. no backwards compatibility).

Reply:
If the user depended on the old behavior, and the behavior changes, that breaks compatibility. Arguably, in a worse way than just breaking at compile-time—that's a lot easier to detect.

Reply:
Got it. I'll specify definitions in `CONTRIBUTING` and modify the PR template to include a few more fields.